index.html

```diff
@@ -0,0 +1,1055 @@
+                  fill="#F3CD03"
+                />
+              </svg>
+              <svg
+                width="20"
+                height="20"
+                viewBox="0 0 20 20"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path
+                  d="M5.03236 12.1308L3.66568 18.0488C3.62336 18.228 3.63607 18.4158 3.70214 18.5877C3.76821 18.7596 3.88457 18.9075 4.03603 19.0122C4.18749 19.1169 4.36701 19.1735 4.55114 19.1746C4.73526 19.1757 4.91544 19.1212 5.06813 19.0183L10.0625 15.6888L15.0568 19.0183C15.213 19.1221 15.3973 19.1755 15.5848 19.1714C15.7723 19.1673 15.954 19.1058 16.1056 18.9953C16.2571 18.8849 16.3712 18.7306 16.4324 18.5534C16.4937 18.3761 16.4992 18.1844 16.4483 18.0039L14.7706 12.1336L18.9312 8.38944C19.0645 8.26946 19.1596 8.11304 19.2049 7.93955C19.2502 7.76605 19.2437 7.58308 19.1861 7.41327C19.1285 7.24345 19.0224 7.09424 18.8809 6.98408C18.7395 6.87392 18.5688 6.80764 18.39 6.79345L13.1609 6.37703L10.8981 1.36801C10.8259 1.20672 10.7086 1.06976 10.5604 0.973657C10.4121 0.877554 10.2392 0.826416 10.0625 0.826416C9.88578 0.826416 9.71286 0.877554 9.56459 0.973657C9.41632 1.06976 9.29902 1.20672 9.22687 1.36801L6.96405 6.37703L1.7349 6.79254C1.55921 6.80645 1.39125 6.87069 1.25111 6.97757C1.11097 7.08445 1.00459 7.22943 0.944699 7.39518C0.884804 7.56094 0.87393 7.74044 0.913378 7.91221C0.952825 8.08398 1.04092 8.24074 1.16714 8.36376L5.03236 12.1308ZM7.64923 8.16288C7.81295 8.14998 7.97019 8.09328 8.10445 7.99871C8.23872 7.90414 8.34508 7.77519 8.41237 7.62538L10.0625 3.97387L11.7126 7.62538C11.7799 7.77519 11.8862 7.90414 12.0205 7.99871C12.1548 8.09328 12.312 8.14998 12.4757 8.16288L16.119 8.45181L13.1187 11.1521C12.8582 11.387 12.7546 11.7484 12.8499 12.0859L13.9992 16.108L10.5725 13.8231C10.422 13.7222 10.245 13.6683 10.0638 13.6683C9.8827 13.6683 9.70565 13.7222 9.55524 13.8231L5.97436 16.2107L6.93746 12.0409C6.97277 11.8876 6.96803 11.7277 6.92369 11.5767C6.87936 11.4257 6.79692 11.2887 6.6843 11.1787L3.89774 8.4619L7.64923 8.16288Z"
+                  fill="#F3CD03"
+                />
+              </svg>
+            </div>
+            <p class="headline headline--small">
+              Slate helps you see how many more days you need to work to reach
+              your financial goal.
+            </p>
+            <div class="card__footer">
+              <img src="/images/client3.png" alt="" class="card__img" />
+              <div class="card__footer-right">
+                <p class="title title--size-small title--color-secondary">
+                  Regina Miles
+                </p>
+                <p class="title title--size-small">Designer</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section id="team" class="team">
```

Comment:
Same with the `id` too. Keep it only if you're using it for a particular functionality, otherwise you don't really need it.